lasagne/layers/conv.py

```diff
@@ -33,29 +32,23 @@ def conv_output_length(input_length, filter_size,
     stride : int
         The stride of the convolution operation.
 
-    border_mode : str, 'valid', 'full', 'same' or 'pad'
-        A string indicating the convolution border mode.
+    pad : int, 'full' or 'same' (default: 0)
+        By default, the convolution is only computed where the input and the
+        filter fully overlap (a valid convolution). When ``stride=1``, this
+        yields an output that is smaller than the input by ``filter_size - 1``.
+        The `pad` argument allows to implicitly pad the input with zeros,
```

Comment:
[You're right!](http://ell.stackexchange.com/questions/11193/allow-to-infinitive-substantive-verb-ing) Both your suggestions are grammatical, mine isn't. I thought "allows to do" and "allows doing" were interchangeable. Will fix the docstring and my [sprachgefühl](https://en.wiktionary.org/wiki/Sprachgef%C3%BChl).